automated install/basic-install.sh

```diff
@@ -1434,99 +1995,139 @@ main() {
     # Clone/Update the repos
     clone_or_update_repos
 
-       # Install packages used by the Pi-hole
-    if [[ ${INSTALL_WEB} == true ]]; then
+    # Install packages used by the Pi-hole
+    if [[ "${INSTALL_WEB}" == true ]]; then
+      # Install the Web dependencies
       DEPS=("${PIHOLE_DEPS[@]}" "${PIHOLE_WEB_DEPS[@]}")
+    # Otherwise,
     else
+      # just install the Core dependencies
       DEPS=("${PIHOLE_DEPS[@]}")
     fi
     install_dependent_packages DEPS[@]
 
+    if [[ -x "$(command -v systemctl)" ]]; then
+      # Value will either be 1, if true, or 0
+      LIGHTTPD_ENABLED=$(systemctl is-enabled lighttpd | grep -c 'enabled' || true)
+    else
+      # Value will either be 1, if true, or 0
+      LIGHTTPD_ENABLED=$(service lighttpd status | awk '/Loaded:/ {print $0}' | grep -c 'enabled' || true)
+    fi
 
     # Install and log everything to a file
     installPihole | tee ${tmpLog}
   else
     # Clone/Update the repos
     clone_or_update_repos
 
-    # Source ${setupVars} for use in the rest of the functions.
+    # Source ${setupVars} for use in the rest of the functions
     source ${setupVars}
 
     # Install packages used by the Pi-hole
-    if [[ ${INSTALL_WEB} == true ]]; then
+    if [[ "${INSTALL_WEB}" == true ]]; then
+      # Install the Web dependencies
       DEPS=("${PIHOLE_DEPS[@]}" "${PIHOLE_WEB_DEPS[@]}")
+    # Otherwise,
     else
+      # just install the Core dependencies
       DEPS=("${PIHOLE_DEPS[@]}")
     fi
     install_dependent_packages DEPS[@]
 
+    if [[ -x "$(command -v systemctl)" ]]; then
+      # Value will either be 1, if true, or 0
+      LIGHTTPD_ENABLED=$(systemctl is-enabled lighttpd | grep -c 'enabled' || true)
+    else
+      # Value will either be 1, if true, or 0
+      LIGHTTPD_ENABLED=$(service lighttpd status | awk '/Loaded:/ {print $0}' | grep -c 'enabled' || true)
+    fi
+
     updatePihole | tee ${tmpLog}
   fi
 
   # Move the log file into /etc/pihole for storage
   mv ${tmpLog} ${instalLogLoc}
 
-  if [[ ${INSTALL_WEB} == true ]]; then
+  if [[ "${INSTALL_WEB}" == true ]]; then
     # Add password to web UI if there is none
     pw=""
+    # If no password is set,
     if [[ $(grep 'WEBPASSWORD' -c /etc/pihole/setupVars.conf) == 0 ]] ; then
+        # generate a random password
         pw=$(tr -dc _A-Z-a-z-0-9 < /dev/urandom | head -c 8)
+        # shellcheck disable=SC1091
         . /opt/pihole/webpage.sh
         echo "WEBPASSWORD=$(HashPassword ${pw})" >> ${setupVars}
     fi
   fi
 
-  echo "::: Restarting services..."
+  echo -e "  ${INFO} Restarting services..."
   # Start services
   start_service dnsmasq
   enable_service dnsmasq
 
-  if [[ ${INSTALL_WEB} == true ]]; then
-    start_service lighttpd
-    enable_service lighttpd
-  fi
+  # If the Web server was installed,
+  if [[ "${INSTALL_WEB}" == true ]]; then
 
-  runGravity
+    if [[ "${LIGHTTPD_ENABLED}" == "1" ]]; then
```

Comment:
Defaults to `0` as `lighttpd` is stopped and disabled by default. (ref line 2011)